Cargo.lock

```diff
@@ -341,7 +341,7 @@ checksum = "437213adf41bbccf4aeae535fbfcdad0f6fed241e1ae182ebe97fa1f3ce19389"
 dependencies = [
  "libc",
  "ndarray",
- "num-complex 0.4.3",
+ "num-complex 0.2.4",
```

Comment:
Is there a reason num-complex got downgraded to 0.2.4, since we had 0.4.3 here before?

Reply:
Fixed it